### PR TITLE
Forum index page - layout and styling

### PR DESCRIPTION
**Pull request by rgilman**
This PR updates the forum index page with a number of changes noted in the commits. It's now a good representation of how I envision that page to look.

### CODE REVIEW

File: assets/styles/style.css [hunk 1]
```diff
@@ -738,7 +738,7 @@ body {
 	font-size: 16px;
 }
 
-#bbpress-forums h3 {
+#bbpress-forums>h3 {
```
Comment (iangilman) [inline assets/styles/style.css]: I think it's easier to read if there are spaces around the `>`.

File: functions/bfc-functions.php [hunk 2]
```diff
@@ -182,3 +182,25 @@ function bfc_filter_bbp_reply_admin_links( $array, $r_id ) {
 
 // add the filter
 add_filter( 'bbp_get_reply_admin_links', 'bfc_filter_bbp_reply_admin_links', 10, 2 );
+
+//removes 'private' and protected prefix for forums
+function remove_private_title($title) {
+	return '%s';
+	}
```
Comment (iangilman) [inline functions/bfc-functions.php]: This curly bracket should go back one indent. Same for line 193.

**Comment by iangilman** [comment]
BTW, you should mention the issue this pull request is addressing, so there's a link between them.

File: assets/styles/style.css (hunk 1 above)
Comment (rgilman) [reply assets/styles/style.css]: Done.

File: functions/bfc-functions.php (hunk 2 above)
Comment (rgilman) [reply functions/bfc-functions.php]: Done.

**Comment by rgilman** [comment]
I fixed the code styling and have done the merge.

**Comment by iangilman** [comment]
Excellent :-)

**Pull request merged**